etcdctl/ctlv3/command/del_command.go

```diff
@@ -70,6 +73,9 @@ func getDelOp(args []string) (string, []clientv3.OpOption) {
 			cobrautl.ExitWithError(cobrautl.ExitBadArgs, fmt.Errorf("too many arguments, only accept one argument when `--prefix` or `--from-key` is set"))
 		}
 		opts = append(opts, clientv3.WithRange(args[1]))
+		if !delRange {
+			fmt.Fprintln(os.Stderr, "Warning: This operations is implicitly deleting range of keys. Please add flag `--range` to make it explicit. Will sleep for a while in v3.6.")
+		}
```

Comment:
I agree with @ahrtr that `Will sleep for a while...` is difficult to interpret. Sorry, I took longer to review, but I would suggest something like these: 
`In etcd v3.6, along with this warning, the delete operation will be suspended for a few seconds to provide the user time to verify range.`    
 or 
`In etcd v3.6, the operation will be suspended for a few seconds to provide the user time to verify range.`
 or 
`In v3.6, the operation will be suspended for a few seconds to provide the user time to verify range.`
Thanks @kkkkun

Reply:
ok, I  would change to the second
`In etcd v3.6, the operation will be suspended for a few seconds to provide the user time to verify range.`

Reply:
Done！